src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -198,7 +202,11 @@ void ChordLayout::layoutPitched(Chord* item, LayoutContext& ctx)
 
             double gapSize = arpeggioNoteDistance;
 
-            if (downnote->line() > firstLedgerBelow || upnote->line() < firstLedgerAbove) {
+            if (leftNote && leftNote->x() == 0) {
+                if (downnote->line() > firstLedgerBelow || upnote->line() < firstLedgerAbove) {
+                    gapSize = arpeggioLedgerDistance + ctx.conf().styleS(Sid::ledgerLineLength).val() * item->spatium();
+                }
+            } else if (leftNote->line() > firstLedgerBelow || leftNote->line() < firstLedgerAbove) {
```

Comment:
Don't we need a null check?
```suggestion
            } else if (leftNote && (leftNote->line() > firstLedgerBelow || leftNote->line() < firstLedgerAbove)) {
```